website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

```diff
@@ -40,3 +40,23 @@ Once you have a job set up to generate documentation for your project, then clic
 ## Easy access
 
 See [Building and Configuring Artifacts](artifacts) for more information on exposing links to the latest Documentation and Source Freshness reports to your team.
+
+## Viewing docs in the IDE
```

Comment:
```suggestion
## Viewing docs
```
This section and the one above describe the same thing. Can you combine them under this heading?

> ## Viewing documentation
> Once you have a job set up to generate documentation for your project, then click **Documentation** in the top left. Your project's documentation should open. This link will always navigate you to the most recent version of your project's documentation in dbt Cloud.
<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/98c05c5-Screen_Shot_2019-02-08_at_9.18.22_PM.png" title="Project documentation"/>